edx_api/enrollments/models_test.py

```diff
@@ -32,6 +32,21 @@ def test_enrollments(self):
         for elem in enrollments_list:
             assert isinstance(elem, Enrollment)
 
+    def test_filter_enrolled_courses(self):
+        """Tests helper method"""
+        with self.assertRaises(ValueError):
+            list(self.enrollments.get_enrolled_courses_by_ids('foo'))
+
+        enrollments_list = list(
+            self.enrollments.get_enrolled_courses_by_ids(('course-v1:edX+DemoX+Demo_Course',))
+        )
+        assert len(enrollments_list) == 1
+        assert enrollments_list[0].course_id == 'course-v1:edX+DemoX+Demo_Course'
+
+    def test_str(self):
+        """Test the __str__"""
+        assert str(self.enrollments) == "<Enrollments>"
```

Comment:
It will be impossible to tell the difference between two enrollments based on their string alone. Not sure if that's a concern, but might see if there's an id that you can include in this.